internal/clients/compliance/client.go

```diff
@@ -44,8 +45,8 @@ func NewClient(conf ComplianceClientConfig) *ComplianceClient {
 	}
 }
 
-func (cc *ComplianceClient) request(ctx context.Context, method, url string) (*http.Response, error) {
-	req, err := http.NewRequestWithContext(ctx, method, url, nil)
+func (cc *ComplianceClient) request(ctx context.Context, method, url string, body io.ReadSeeker) (*http.Response, error) {
```

Comment:
This could just be an `io.Reader`, as we're not seeking. But it doesn't matter really.